index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- displays site properly based on user's device -->
+
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="./assets/images/favicon-32x32.png"
+    />
+
+    <title>Frontend Mentor | Mortgage repayment calculator</title>
+
+    <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
+    <style>
+      .attribution {
+        font-size: 11px;
+        text-align: center;
+      }
+      .attribution a {
+        color: hsl(228, 45%, 44%);
+      }
+    </style>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <!-- Form -->
+
+      <section class="form__container">
+        <h1>Mortgage Calculator</h1>
+
+        <form id="mortgageForm" class="mortgageForm" method="POST">
+          <input type="reset" value="Clear All" class="input-reset" />
+
+          <label for="amount">Mortgage Amount</label>
+          <div class="input-box">
+            <span class="input-icon">£</span>
+            <input
+              type="number"
+              id="amount"
+              name="amount"
+              step="0.01"
+              required
+            />
```

Comment:
Remove this `required` from every form input to show the javaScript error message and please correct the styling